core/src/main/java/com/github/szsalyi/flashcard/entity/BaseEntity.java

```diff
@@ -0,0 +1,27 @@
+package com.github.szsalyi.flashcard.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+import java.io.Serializable;
+
+/**
+ * Created by ssalyi on 8/14/2017.
+ */
+
+
+@MappedSuperclass
+public class BaseEntity implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Getter
```

Comment:
Instead of specifying the `@Getter` and `@Setter` annotations on you class fields, you can just annotate the class itself with `@Data`, which will do the same for every field.